lib/random/formatter.rb

```diff
@@ -1,44 +1,57 @@
 # -*- coding: us-ascii -*-
 # frozen_string_literal: true
 
-# == Random number formatter.
+# == \Random number formatter.
 #
-# Formats generated random numbers in many manners.
+# Formats generated random numbers in many manners. When <tt>'random/formatter'</tt>
+# is required, the module is mixed in the core class Random and standard
+# library SecureRandom, and module's methods are becoming available in them.
```

Comment:
More accurately:
1. `Random::Formatter` is defined in random.c and mixed in `Random`, and random/formatter.rb add some method to the module.
2. `SecureRandom` module object itself is `extend`ed, not mixing in as instance methods.

Reply:
Thanks! I updated the docs.